udata/core/dataset/api_fields.py

```diff
@@ -106,6 +106,9 @@
     'frequency': fields.String(
         description='The update frequency', required=True,
         enum=UPDATE_FREQUENCIES.keys(), default=DEFAULT_FREQUENCY),
+    'frequency_date': fields.ISODateTime(
+        description=('Future date of update, you will be notified '
```

Comment:
Maybe 'Next expected update date' instead of 'Futur date of update'. What do you think about ?